syncstorage/tests/test_wsgiapp.py

```diff
@@ -109,20 +112,7 @@ def test_logging_of_invalid_bsos(self):
             assert False, "log was not generated"
 
     def test_metrics_capture_for_batch_uploads(self):
-        app = TestApp(self.config.make_wsgi_app())
-
-        # Monkey-patch the app to make legitimate hawk-signed requests.
-        user_id = 42
-        auth_policy = self.config.registry.getUtility(IAuthenticationPolicy)
-        req = Request.blank("http://localhost/")
-        auth_token, auth_secret = auth_policy.encode_hawk_id(req, user_id)
-
-        def new_do_request(req, *args, **kwds):
-            hawkauthlib.sign_request(req, auth_token, auth_secret)
-            return orig_do_request(req, *args, **kwds)
-
-        orig_do_request = app.do_request
-        app.do_request = new_do_request
```

Comment:
As far as I can tell this was just duplicated code from the `make_test_app` method, so I cleaned it up while I was in here.